### DISCUSSION TITLE
Does someone know the default values for Thermal Foundation and Tinker's Construct ores? (1.16.5)

### DISCUSSION THREAD

**Discussion by PiBombbb** [EwyBoy/OreTweaker#83] [Help Wanted!] [answered]
Basically the title. I want to make all ores twice as easy to find so I have an easier time in my 1.16.5 custom modpack but the wiki only contains default information for vanilla ores.

Thanks. Also I apologize if I posted this in the wrong place.

**Comment by EwyBoy** [comment, marked as answer]
Thermal Expansion is closed source. So there is no legal way at looking up the values in the mods world generation.
You either have to ask the developers or try to replicate it as close as possible.
